app/src/ui/changes/no-changes.tsx

```diff
@@ -280,9 +280,9 @@ export class NoChanges extends React.Component<
 
     const description = (
       <>
-        Configure which editor you wish to use in{' '}
+        Select your editor{' '}
```

Comment:
I think this should do it but feel free to reject the suggestion if not.

```suggestion
        Select your editor in {' '}
```

Reply:
Someone else will need to commit, as I do not have permissions.